skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
Yea, this flag only states if we want the user to see a DVR UI if and only if the stream is DVR capable. If it is disabled/false, then it doesn't matter if the stream comes with DVR, as we won't show any UI for it.